pkgs/by-name/ke/keychron-udev-rules/package.nix

```
@@ -0,0 +1,39 @@
+{
+  writeTextFile,
+  lib,
+  stdenvNoCC,
+  udevCheckHook,
+
+  idProduct ? "0161", # Keychron Q6 Keyboard Product Id
+}:
+stdenvNoCC.mkDerivation (final: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Convention is to use `finalAttrs`.